podium/settings.py

```diff
@@ -82,8 +82,7 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
Please do not change the default database configuration.  Instead, use your own local environment configuration as described in https://github.com/pyatl/podium-django/blob/master/README.md#usage